.devops/nix/package.nix

```diff
@@ -1,35 +1,37 @@
-{
-  lib,
-  config,
-  stdenv,
-  mkShell,
-  cmake,
-  ninja,
-  pkg-config,
-  git,
-  python3,
-  mpi,
-  openblas, # TODO: Use the generic `blas` so users could switch between alternative implementations
-  cudaPackages,
-  darwin,
-  rocmPackages,
-  vulkan-headers,
-  vulkan-loader,
-  clblast,
-  useBlas ? builtins.all (x: !x) [
+{ lib
+, config
+, stdenv
+, mkShell
+, cmake
+, ninja
+, pkg-config
```

Comment:
The previous commit used `github:piegamesde/nixfmt/commits/rfc101-style` for formatting, let's keep this consistent

Reply:
Thanks, I was wondering how styles are managed. Should we perhaps add a `formatter` to flake.nix?

Reply:
Probably a good idea. A separate PR maybe?

Reply:
Will open one soon

Reply:
#5687